Cannot find module 'vue-template-compiler'

**Describe the bug**
Running `yarn test --watch` results in the following:

```
  ● Test suite failed to run

    Cannot find module 'vue-template-compiler'
    Require stack:
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/vue-jest/lib/process.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/vue-jest/vue-jest.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-util/build/requireOrImportModule.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-util/build/index.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-config/build/getCacheDirectory.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-config/build/Defaults.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-config/build/normalize.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-config/build/index.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-cli/build/init/index.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-cli/build/cli/index.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-cli/build/index.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest-cli/bin/jest.js
    - /Users/stuartclark/Projects/Druxt/druxt-auth/node_modules/jest/bin/jest.js

      at Object.<anonymous> (node_modules/vue-jest/lib/process.js:1:21)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create a new module from this template.
2. Install and run tests: `yarn && yarn test`
3. See error

**Expected behavior**
Run tests

**Media**
N/A

**Your Environment (please complete the following information):**
N/A

**Additional context**
N/A

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "eslint-plugin-nuxt": "3.1.0",
     "jest": "27.5.1",
     "siroc": "0.16.0",
-    "vue-jest": "3.0.7"
+    "vue-jest": "3.0.7",
+    "vue-template-compiler": "^2.6.14"
   }
 }

